src/data/accessibility-quiz.ts

```diff
@@ -278,6 +278,17 @@ const accessibilityQuiz = [
       "The aria-pressed attribute indicates the current pressed state of a toggle button.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed"
   },
+  {
+    Question:
+      "What is the semantic HTML equivalent for the ARIA: link role (role=`link`)?",
+    Answer: "<a>",
+    Distractor1: "<nav>",
+    Distractor2: "<link>",
+    Distractor3: "<iframe>",
+    Explanation:
+      "Where possible, it is recommended that you use a native <a> element rather than the link role, as native elements are more widely supported by user agents and assistive technology.",
```

Comment:
Please do not use direct quotes from external sites like MDN. It is always best to write the explanations in your won words 👍 

```suggestion
      "The anchor element is the HTML equivalent to the  ARIA link role. It is best to use the anchor element because it is best supported by assistive technologies.",
```

Reply:
Made changes as per your comments. Please review.